### PR TITLE
completed set up ticket for this project

### PR DESCRIPTION
**Pull request by dmarkesini**
Have followed instructions on the set up ticket to set up my project which includes:
- updating the README file,
- creating an index file inside the data folder to export both the individual data sets (dev and test)
-creating the .env. files
- had a look at my scripts and installed necessary packages
- then proceeded to complete the set up by creating the app.js file, controller, model folders and their corresponding files, the listen.js file

### CODE REVIEW

File: db/data/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const devData = require(`./development-data/index`);
+const testData = require(`./test-data/index`);
+
+const ENV = process.env.NODE_ENV || "development";
+
+const data = { development: devData, test: testData };
+
+module.exports = data[ENV];
```
Comment (karlaferry) [inline db/data/index.js:8]: I think this file would be unnecessary because connection.js is already setup for you.

**Review by karlaferry — APPROVED** [review]
Approooooved :)

File: db/data/index.js (hunk 1 above)
Comment (karlaferry) [reply db/data/index.js:8]: Apologies. Just realised what you were trying to do here. I think it should work! Any problems, let us know!

Comment (dmarkesini) [reply db/data/index.js:8]: Okay thank you! the ticket said this: " Make sure the index file exports an object with values of the data from that folder " so the only file where we export the data into an object is the index one in the data folder, unless I misunderstood! Either way thank you for the comments!

**Pull request merged**